Modules/System/Azure Blob Services API/src/ABSBlobClient.Codeunit.al

```diff
@@ -73,6 +73,31 @@ codeunit 9053 "ABS Blob Client"
         exit(ABSClientImpl.ListBlobs(ABSContainerContent, ABSOptionalParameters));
     end;
 
+    /// <summary>
+    /// Lists the blobs in a specific container.
+    /// see: https://go.microsoft.com/fwlink/?linkid=2210588
+    /// </summary>    
+    /// <param name="BlobList">Collection of the result (BlobList of [Text, XmlNode]).</param>
```

Comment:
Could you please add more details about the BlobList Result. e.g. what is the key of the dictionary.

Reply:
Is now updated